src/test/java/io/openshift/booster/OpenShiftIT.java

```diff
@@ -76,4 +65,84 @@ public void testGreetingEndpointWithNameParameter() {
                 .body("content", is("Hello John from a ConfigMap!"));
     }
 
+    @Test
+    public void testCConfigMapUpdate() {
+        verifyEndpoint();
+        updateConfigMap();
+        rolloutChanges();
+        waitForApp();
+        when().get()
+                .then()
+                .statusCode(200)
+                .body("content", is("Bonjour World from a ConfigMap!"));
+    }
+
+    @Test
+    public void testDConfigMapNotPresent() {
+        get().then()
+                .statusCode(200);
+        assistant.client()
```

Comment:
Since ConfigMap update was extracted to the separate method, it would be consistent to extract delete too.